frontend/src/App.tsx

```diff
@@ -17,9 +17,10 @@ function App() {
     <div className="app-container">
       <Routes>
         <Route path="/" element={<Layout />}>
-          <Route index element={<Candidates />} />
-          <Route path="candidates" element={<Candidates />} />
+          <Route index element={<Home />} />
+          <Route path="home" element={<Home />} />
           <Route path="apply" element={<Apply />} />
+          <Route path="candidates" element={<Candidates />} />
```

Comment:
I should have notice this earlier but lets call the `<home>` element candidates as it makes more sense. I feel like home sounds misleading in making it sould like the homepage of the website and `<candidates>` makes more sense but its not. Let me know if you think other wise. Of candidates and home we only need 1 since they are essentially the same page.

Reply:
are you saying to rename the Home page as the Candidates page?

Reply:
yeah